app/controllers/employees_controller.rb

```diff
@@ -76,7 +62,6 @@ def destroy
     # Use callbacks to share common setup or constraints between actions.
     
     # Before filters
-  
     def set_employee
```

Comment:
As we discussed before, you may want to avoid using before_action setup for things like this.